core/controllers/feedback.py

```diff
@@ -63,9 +63,13 @@ class ThreadHandler(base.BaseHandler):
 
     def get(self, exploration_id, thread_id):  # pylint: disable=unused-argument
         suggestion = feedback_services.get_suggestion(exploration_id, thread_id)
+        messages = [m.to_dict() for m in feedback_services.get_messages(
+            exploration_id, thread_id)]
+        message_ids = [message['message_id'] for message in messages]
+        feedback_services.update_messages_read_by_the_user(
```

Comment:
Oh -- maybe I misunderstand. Is this supposed to be update_messages_shown_to_the_user()? ("Shown to" is not the same as "read by".)

That said, isn't "read by" what you actually care about here?